### DISCUSSION TITLE
About their approximate memory requirements and relative speed.

### DISCUSSION THREAD

**Discussion by kimdwkimdw** [openai/whisper#535] [Q&A] [answered]
https://github.com/openai/whisper#available-models-and-languages

Relative speed can vary in a kind of GPU. What kinds of GPU in Table?

**Comment by glangford** [comment, marked as answer]
These are very approximate relative speeds in the Table, and performance will vary widely depending on the language and the settings. Since you haven't received an answer yet, you might watch https://github.com/openai/whisper/discussions/480 and https://github.com/openai/whisper/discussions/530 .
